### PR TITLE
Auto code surveys

### PR DESCRIPTION
**Pull request by as2388**
Answer latest comments on #3 and review #4 first.

Implemented gender only. If this looks good will do the rest.

### CODE REVIEW

**Comment by lukechurch** [comment]
I think I probably need a merge from #3 to be able to review this

**Comment by as2388** [comment]
@lukechurch This can be reviewed now

File: project_redss/lib/dataset_specification.py [hunk 1]
```diff
@@ -22,35 +23,49 @@ class DatasetSpecification(object):
...
-                   code_translator=S01E01Translator),
+                   code_scheme=CodeSchemes.S01E01),
 
         CodingPlan(raw_field="rqa_s01e02_raw",
                    coded_field="rqa_s01_e02_coded",
                    time_field="sent_on",
                    coda_filename="s01e02",
                    cleaner=None,
-                   code_translator=S01E01Translator),  # TODO: Use S01E02 when available
+                   code_scheme=CodeSchemes.S01E01),  # TODO: Use S01E02 when available
```
Comment (lukechurch) [inline project_redss/lib/dataset_specification.py:35]: This seems a bit error prone - could we put something in that breaks if we accidentally use this before it's fixed?

File: project_redss/lib/redss_schemes.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import json
+
+from core_data_modules.data_models import Scheme
+
+
+def open_scheme(filename):
```
Comment (lukechurch) [inline project_redss/lib/redss_schemes.py]: _open_scheme?

**Review by lukechurch — APPROVED** [review]

**Pull request merged**